### ISSUE TITLE
Can you translate you README

### ISSUE THREAD

**Issue by olaferlandsen** [thinkanyforever/electron-xp#1]
and add the source code.
It can work with Windows XP SP2?

**Comment by thinkanyforever** [comment]
@olaferlandsen yes, It can work with Windows XP SP2, but the source code is too large, do you need a binary version?

**Comment by olaferlandsen** [comment]
> `@olaferlandsen yes, It can work with Windows XP SP2, but the source code is too large, do you need a binary version?`

No, I think on sourcecode. I like opensource ;)
